app/Http/Controllers/GroupController.php

```diff
@@ -23,6 +25,34 @@ public function __construct()
         $this->beforeFilter('auth', array('on' => array('post', 'put', 'delete')));
     }
 
+    /**
+     * Store newly created Group.
+     *
+     * @return Response
+     */
+    public function store(StoreGroupRequest $request)
```

Comment:
This is part of the Form Request Object pattern that Laravel uses. The type hinting tells it to use the file named `StoreGroupRequest.php` in the `app/Http/Requests/` folder and will apply the authorization and validation logic in it.